### DISCUSSION TITLE
Activate approach fase

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#7043] [Help] [answered]
I tried to google, to see when I've to activate approach fase. In most cases the automatic activation is way to late (passing the circled D), is that a bug or?

Is there a distance or altitude what is used to start activation?

**Comment by ExampleWasTaken** [comment, marked as answer]
You activate it whenever it is convenient for you ;)
Usually when your speed gets close to green dot speed during approach. The model is realistic as most pilots activate the approach phase manually because of that. 

For further questions I can only recommend the YouTube channel of [320 Sim Pilot](https://youtube.com/@320SimPilot)! He's got a lot of very detailed and high quality videos and pretty much all normal, as well as some abnormal operations.

**Reply by joeherwig** [reply]
Maybe you wanna check out what Airbus says:

I would like to share the following media published on Airbus WIN: 
https://www.airbus-win.com/wp-content/uploads/2019/01/what-about-arming-conditions-of-appr-modes.mp4

Joe

**Reply by ExampleWasTaken** [reply]
That is essentially what I said. 
> Usually when your speed gets close to green dot speed during approach.

**Reply by tracernz** [reply]
The automatic activation should be at the latest possible point to still arrive at Vapp at 1000 feet above the runway (same for flap PWPs when the CDA option is enabled). If you are used to configuring earlier than the last possible opportunity that will feel late.
